runit-ruby-bionic/README.md

```diff
@@ -0,0 +1,26 @@
+socrata/runit-ruby-bionic
+============
+
+socrata/runit-bionic image with Ruby installed
+
+### Usage
+
+Most uses of the image will be via `FROM socrata/runit-ruby-bionic:<version>` in a Dockerfile, nonetheless, you can run a Ruby container as follows:
+
+    $ docker pull socrata/runit-ruby-bionic:<version>
+
+    # Launch shell in the container
+    $ docker run --rm -t -i socrata/runit-ruby-bionic:<version> bash
+
+    # Bind mount a directory into the container and build or run something
+    $ docker run --rm -t -i -v `pwd`:/opt/my_app socrata/runit-ruby-bionic:<version> ruby my_app.rb
+
+
+### Available versions
+
+- `socrata/runit-ruby-bionic` _alias for `socrata/runit-ruby-bionic:2.7.2`_
```

Comment:
I know I might be a bit late to the party on some discussions, but is there any reason not to make the default "bionic" 3.0.0?

Reply:
2.7.2 is all that was actually in the scope of this project officially. Marc Slemco requested I add 3.0.0 when doing the ruby-bionic image updates. It was quick and easy to do add ruby 3, but I think the current plan is to get everything moved from 2.5 to 2.7.